controllers/component_build_controller_test.go

```diff
@@ -164,6 +164,88 @@ var _ = Describe("Component initial build controller", func() {
 			waitComponentAnnotationValue(resourceKey, PaCProvisionAnnotationName, PaCProvisionDoneAnnotationValue)
 		})
 
+		It("should fail to submit PR if GitHub application is not installed into repo", func() {
```

Comment:
Interesting. But, the "app is not installed into repo" failure branch is not covered.

Reply:
hmm, how about keep this test because it's for a specific test purpose? That test you mentioned is actually for another test purpose and in theory, a failure prerequisite can be various not only "app is not installed into a repo".

Reply:
Up to you, I don't have strong opinion on this.